templates/coderd.yaml

```diff
@@ -23,7 +27,9 @@ spec:
   template:
```

Comment:
unfortunately, matchLabels is immutable, so we can't add the new labels here.

there might be interesting behavior if someone installs two separate releases coder deployments into the same namespace, but I guess that's not possible since the names would conflict anyways